docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -27,6 +27,7 @@ The :ref:`porting_2.10_guide_base` is included in this porting guide. The comple
   :local:
   :depth: 2
 
+
```

Comment:
The changes to this file should be omitted since this file is automatically generated now. I didn't know that at the time, so the text here is redundant.